Sources/SPMBuildCore/PluginInvocation.swift

```diff
@@ -506,35 +587,35 @@ public protocol PluginInvocationDelegate {
     func pluginRequestedSymbolGraph(forTarget name: String, options: PluginInvocationSymbolGraphOptions, completion: @escaping (Result<PluginInvocationSymbolGraphResult, Error>) -> Void)
 }
 
-public struct PluginInvocationSymbolGraphOptions: Decodable {
```

Comment:
As with the PackagePlugin API, the PluginMessages file now contains all the structs that get serialized over the wire, and so these structures are only client-facing now and there is no coupling to the wire protocol.